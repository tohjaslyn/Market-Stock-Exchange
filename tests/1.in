BALANCE trader1
INVENTORY trader1
AMOUNT trader1 ABC
SELL trader1 ABC 10 1.5
BUY trader1 ABC 12.5 1
IMPORT trader1 ABC 100
EXPORT trader1 ABC 1.4
CANCEL SELL 0000
CANCEL BUY 000A
ORDER AFEC
TRADERS
TRADES
TRADES TRADER trader1
TRADES PRODUCT ABC
BOOK SELL
BOOK BUY
EXIT
Have a nice day.
