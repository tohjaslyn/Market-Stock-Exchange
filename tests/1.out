$ No such trader in the market.
$ No such trader in the market.
$ No such trader in the market.
$ No such trader in the market.
$ No such trader in the market.
$ No such trader in the market.
$ No such trader in the market.
$ No such order in sell book.
$ No such order in buy book.
$ No orders in either book in the market.
$ No traders in the market.
$ No trades have been completed.
$ No such trader in the market.
$ No trades have been completed with given product.
$ The sell book is empty.
$ The buy book is empty.
$ Have a nice day.
